pkg/didcomm/envelope/jwe/authcrypt/encrypt_jwk_test.go

```diff
@@ -66,7 +66,7 @@ func TestNilEncryptSenderJwk(t *testing.T) {
 	require.Empty(t, l)
 	require.Empty(t, m)
 
-	pld, err := crypter.Encrypt([]byte(""), someKey[:], [][]byte{someKey[:]})
+	pld, err := crypter.Pack([]byte(""), someKey[:], [][]byte{someKey[:]})
```

Comment:
why is the variable name still `crypter` ?

Reply:
I'll go over all other uses of 'crypter' too.

Reply:
this should be done now.